config/urls.yaml

```diff
@@ -226,7 +226,7 @@
   video:
     isc: https://innersourcecommons.org/learn/learning-path/contributor/02/
     oreilly: https://learning.oreilly.com/learning-paths/learning-path-the/0636920338833/0636920338802-video329500
-    youtube: https://youtu.be/S0Gps2AbZ7M
+    youtube: https://www.youtube.com/watch?v=S0Gps2AbZ7M
```

Comment:
Just curious why this change?

Reply:
When this was updated in #431 it used the `youtu.be` format, but the script expects `www.youtube.com` format - all other YouTube URLS in this file are in that format. The script needs to extract the video ID from the URL.